### PR TITLE
Improve helm installation instructions

### LINKED ISSUE #3587: Inconsistent helm install page
for 1.2, I'd like us to have some consistency for the helm install page

We have 2 helm options (template vs helm install).   one requires downloading istio binary, one doesn't but asks users to do helm fetch.

Then at the end we ask users to do helm fetch (which some of them already did earlier) to clean up CRDs. (https://preliminary.istio.io/docs/setup/kubernetes/install/helm/#deleting-crds-and-istio-configuration)

We need some consistency on this page:  https://preliminary.istio.io/docs/setup/kubernetes/install/helm

### PR DESCRIPTION
**Pull request by frankbu**
Fixes #3587

### CODE REVIEW

File: content/docs/setup/kubernetes/install/helm/index.md [hunk 1]
```diff
@@ -158,6 +173,24 @@ $ helm template $HOME/istio-fetch/istio --name istio --namespace istio-system \
...
+{{< text bash >}}
+$ helm template $HOME/istio-fetch/istio --name istio --namespace istio-system \
+      --values $HOME/istio-fetch/istio/values-istio-sds-auth.yaml | kubectl apply -f -
+{{< /text >}}
+
+{{% /tab %}}
+
 {{< /tabset >}}
 
 ### Option 2: Install with Helm and Tiller via `helm install`
```
Comment (ericvn) [inline content/docs/setup/kubernetes/install/helm/index.md]: My problem with Option #2 and the way it's laid out is that certain commands like the tiller command, `kubectl apply -f @install/kubernetes/helm/helm-service-account.yaml@` require that one has downloaded the Istio release. There is no way to get the yaml file from the helm repository (at least that's what I was told yesterday).

So, we need instructions or a pointer to that: https://preliminary.istio.io/docs/setup/kubernetes/download/

That then loads to the question of whether we want to use the downloaded repository or the helm repository for the commands. We would need to put the various `fetch` commands in this section, or I think we should just update the commands in the tab to use the local repo.

They would be on the order of:

`helm install install/kubernetes/helm/istio-init --name istio-init --namespace istio-system`

and in the tabs:
`helm install install/kubernetes/helm/istio --name istio --namespace istio-system`

`helm install install/kubernetes/helm/istio --name istio --namespace istio-system --values install/kubernetes/helm/istio/values-istio-demo.yaml`

`helm install install/kubernetes/helm/istio --name istio --namespace istio-system --values install/kubernetes/helm/istio/values-istio-minimal.yaml`

`helm install install/kubernetes/helm/istio --name istio --namespace istio-system --values install/kubernetes/helm/istio/values-istio-remote.yaml`

File: content/docs/setup/kubernetes/additional-setup/config-profiles/index.md [hunk 2]
```diff
@@ -14,21 +14,29 @@ your specific needs. The following built-in configuration profiles are currently
...
+    {{< /warning >}}
 
 1. **minimal**: the minimal set of components necessary to use Istio's [traffic management](/docs/tasks/traffic-management/) features.
 
-1. **remote**: creates a service account with minimal access for use by Istio Pilot discovery used in [configuring a multicluster mesh](/docs/examples/multicluster/split-horizon-eds/).
+1. **remote**: used for configuring remote clusters of a
+    multicluster mesh with a [single control plane topology](/docs/concepts/multicluster-deployments/#single-control-plane-topology).
 
-1. **sds**:  used to enable [SDS (secret discovery service) for Istio](/docs/tasks/security/auth-sds). This profile comes only with authentication enabled.
+1. **sds**: similar to the **default** profile, but also enable Istio's [SDS (secret discovery service)](/docs/tasks/security/auth-sds).
```
Comment (ericvn) [inline content/docs/setup/kubernetes/additional-setup/config-profiles/index.md]: c/enable/enables/

File: content/docs/setup/kubernetes/install/helm/index.md [hunk 3]
```diff
@@ -50,21 +56,12 @@ the appropriate snapshot.  For example, if you want to run with snapshot 6, use
 [URL](https://gcsweb.istio.io/gcs/istio-prerelease/prerelease/1.1.0-snapshot.6/charts) in installation step 1 below.
 {{< /tip >}}
 
-1.  Update Helm's local package cache with the location of the Helm daily release:
+1. Update Helm's local package cache with the location of the Helm daily release:
```
Comment (ericvn) [inline content/docs/setup/kubernetes/install/helm/index.md]: Based on what I said below, I would move this and the working directory creation along with the fetch inside Option #1.

File: content/docs/setup/kubernetes/install/helm/index.md (hunk 1 above)
Comment (ericvn) [reply content/docs/setup/kubernetes/install/helm/index.md]: Although I do see that using the `fetched` would possibly limit the number of changes. Just more difficult getting tiller ready. I recall we used to have a kubectl apply of the file contents pasted in the doc.

File: content/docs/setup/_index.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Setup and Upgrade
```
Comment (geeknoid) [inline content/docs/setup/_index.md]: Why remove Upgrade?

File: content/docs/setup/_index.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
...
+title: Setup
 description: How to deploy and upgrade Istio in various environments, for example Kubernetes and Consul.
```
Comment (geeknoid) [inline content/docs/setup/_index.md]: how about

"...environments such as Kubernetes and Consul".

File: content/docs/setup/kubernetes/install/helm/index.md (hunk 1 above)
Comment (frankbu) [reply content/docs/setup/kubernetes/install/helm/index.md]: I put it in the doc for now, but we should come up with a better solution in the future. Note that users will need to download Istio to install the `istioctl` CLI, so we need a better way for that too if we want to be able to use Istio without downloading the release.

**Comment by geeknoid** [comment]
See related discussion in #3325

File: content/docs/setup/_index.md (hunk 4 above)
Comment (frankbu) [reply content/docs/setup/_index.md]: All the other top level items are one word, so this one seems out of place. The way I look at it, at a high level it's all just part of setting up Istio. We don't mention Install at this level either. `Download, Install, and Upgrade` would be a better title if we did.

File: content/docs/setup/_index.md (hunk 5 above)
Comment (frankbu) [reply content/docs/setup/_index.md]: done

File: content/docs/setup/kubernetes/additional-setup/config-profiles/index.md (hunk 2 above)
Comment (frankbu) [reply content/docs/setup/kubernetes/additional-setup/config-profiles/index.md]: done

**Pull request merged**